recipe/meta.yaml

```
@@ -23,48 +23,56 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
-    - babel
+    - flit-core >=3.7
   run:
     - python
-    - alabaster >=0.7,<0.8
-    - babel >=1.3
-    - docutils >=0.14,<0.19
-    - imagesize
-    - importlib-metadata >=4.4  # [py<310]
-    - jinja2 >=2.3
+    - alabaster >=0.7.14,<0.8
+    - babel >=2.9
+    - docutils >=0.18.1,<0.22
+    - imagesize >=1.3
+    - importlib-metadata >=4.8  # [py<310]
+    - jinja2 >=3.0
     - packaging
-    - pygments >=2.0
+    - pygments >=2.14
     - requests >=2.5.0
```

```suggestion
    - requests >=2.25.0
```
https://github.com/sphinx-doc/sphinx/blob/v7.3.7/pyproject.toml#L72